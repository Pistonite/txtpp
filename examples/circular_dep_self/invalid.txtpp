#include directives cannot include the same file
TXTPP#include invalid
It will still try to process it. The first line will be emitted, but not this line.