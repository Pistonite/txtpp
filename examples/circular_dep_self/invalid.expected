#include directives cannot include the same file